website_sale_catseo/README.md

```diff
@@ -0,0 +1,8 @@
+Public Product Category CMS Block
+=================================
+
+Make it possible to add a cms block on top and buttom to public product categories in product block and list views.
```

Comment:
maybe you could explain what a cms block is? I'm not a specialist, but for me cms == content management system, and it sounds strange here.